tcf_website/models/models.py

```diff
@@ -379,6 +381,31 @@ class Course(models.Model):
     def __str__(self):
         return f"{self.subdepartment.mnemonic} {self.number} | {self.title}"
 
+    def course_description_without_prerequisite(self):
+        """Returns course description without its pre-requisite(s)."""
+        return self.description.replace(self.pre_req, "")
+
+    def prerequisite(self):
```

Comment:
There is most definitely a less line heavy way of implementing this: open to suggestions.